apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/Session.cs

```diff
@@ -0,0 +1,175 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Google.Apis.Auth.OAuth2;
+
+namespace Google.Cloud.Spanner.V1
+{
+    public partial class Session : IDisposable
+    {
+        private readonly CancellationTokenSource _cancellation = new CancellationTokenSource();
+        private Task _keepAlive;
+        private int _lastAccessTime = Environment.TickCount;
```

Comment:
Agree with @jskeet about using time-based types in all possible places, unless there's a *very* good reason not to. We've consistently done this in all existing code in this repo.
I accept that this is more important when publicly exposed, but I'd much rather see time-based types used internally as well.
If code *must* use ints/longs, then please always put the unit in the name.